sublime-phpunit.py

```diff
@@ -119,7 +119,7 @@ def find_project_root(self, file_name):
         found = False
         while found == False:
             file_path = os.path.abspath(os.path.join(file_path, os.pardir))
-            found = os.path.isfile(file_path + '/artisan') or file_path == '/'
+            found = os.path.isfile(file_path + '/phpunit.xml') or file_path == '/'
```

Comment:
Well, not sure switching to phpunit.xml is a good choice, because of the projects which use `phpunit.xml.dist` or something similar. Maybe searching for `.git` would be more reliable?